### PR TITLE
[LIB-646] add geo point support to LIB;

### PR DESCRIPTION
**Pull request by opalczynski**
Will add test for it when end up the filters :)

### CODE REVIEW

**Comment by lifcio** [comment]
That was quick :)

**Comment by 23doors** [comment]
@opalczynski filters have been merged today as well (already on stg)

**Comment by opalczynski** [comment]
@lifcio Well I have a task in this sprint to handle geo point from parse :) So firstly must do this here :) Thanks for task creation.

@23doors Ok, thanks, will handle it, but probably at the end of the next week.

File: syncano/models/fields.py [hunk 1]
```diff
@@ -679,6 +681,66 @@ def to_native(self, value):
...
+                raise SyncanoValueError('Expected an object')
+
+        if not isinstance(value, GeoPointStruct):
+            raise SyncanoValueError('Expected an GeoPointStruct')
+
+    def to_native(self, value):
+        if value is None:
+            return
+
+        geo_struct = {'longitude': value[0], 'latitude': value[1]}
```
Comment (dancio) [inline syncano/models/fields.py]: this will blow up in case of `value = {}`

**Pull request merged**